configuration.go

```diff
@@ -46,10 +46,12 @@ const (
 	BundlePolicyMaxCompat
 )
 
-var BundlePolicyString = []string{"Balanced", "MaxBundle", "MaxCompat"}
-
 func (p BundlePolicy) String() string {
-	return EnumToStringSafe(int(p), BundlePolicyString)
+	return EnumToStringSafe(int(p), []string{
+		"Balanced",
```

Comment:
I was also going to comment that the compiler would likely optimize away such constant slices (allocate on the stack, etc), but I was unsure.  Now I've found a thread – and an old one, at that (I'm even more optimistic about the new SSA-based implementation) – that makes it pretty clear such optimizations really are done:

https://groups.google.com/forum/#!topic/golang-nuts/KdbtOqNK6JQ

Cool, huh?